server/command.go

```diff
@@ -195,6 +199,7 @@ func (p *Plugin) executeLinkCommand(args *model.CommandArgs, parameters []string
 		Creator:             args.UserId,
 	}
 
+	p.sendBotEphemeralPost(args.UserId, args.ChannelId, commandWaitingMessage)
```

Comment:
why this has been added

Reply:
I think this should be in a go routine maybe ?

Reply:
discussed